components/common/src/command/package/config.rs

```diff
@@ -30,11 +31,19 @@ use toml;
 
 use error::Result;
 
-pub fn start(ident: &PackageIdent, fs_root_path: &Path) -> Result<()> {
-    let package = try!(PackageInstall::load(ident, Some(fs_root_path)));
+pub fn start<P: ?Sized>(ident: &PackageIdent, fs_root_path: &P) -> Result<()>
```

Comment:
You actually don't need to take a reference on `P` here since you're asking for P to implement the trait `AsRef<Path>`. You can just pass `Some(fs_root_path.as_ref())` to `PackageInstall::load` and you're good to go!